data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -54,7 +57,9 @@ def _extract_counties(self, county_path: Path) -> pd.DataFrame:
     def _extract_states(self, state_path: Path) -> pd.DataFrame:
         logger.info("Reading States CSV")
         return pd.read_csv(
-            state_path, dtype={"fips": "string", "state_abbreviation": "string"}
+            state_path,
```

Comment:
i think we can just read in the columns that we want to read, rather than dropping columns in place.

Reply:
Nice! You're an excellent campsite cleaner. I really appreciate this about your work.